src/dashboard/src/components/api/views.py

```diff
@@ -55,6 +55,13 @@
     r"^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$",
     re.IGNORECASE,
 )
+JOB_STATUS_CODE_LABELS = {
+    models.Job.STATUS_UNKNOWN: "UNKNOWN",
+    models.Job.STATUS_AWAITING_DECISION: "USER_INPUT",
+    models.Job.STATUS_COMPLETED_SUCCESSFULLY: "COMPLETE",
+    models.Job.STATUS_EXECUTING_COMMANDS: "PROCESSING",
+    models.Job.STATUS_FAILED: "FAILED",
+}
```

Comment:
Hmm. I used the same labels because I kind of thought about it when I created this mapping. However the intended use seemed different. It's like in this mapping there's a 1-1 relationship between the *STATUS* of the job and the label. In the `get_unit_status` function I don't see that 1-1 relationship. The label returned is based on very different conditions and you have additional ones like `REJECTED` not based on status at all. Not sure how to express this point. Sorry about that.

Reply:
Hmm, one to ponder. Definitely worth reflecting on the semantics before changing something for the sake of removing small potential redundancy. We still need the other end point to be consistent where this one can change too. And we can still make this change in the future if the two sets of values look closely aligned.